.github/workflows/tests.yml

```diff
@@ -24,13 +24,19 @@ jobs:
           - 3.8
           - 3.9
           - "3.10"
+        exist-version:
+          - "4.10.0"
+          - "5.4.1"
+          - "6.0.1"
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ inputs.ref || github.ref }}
       - uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Set EXIST_VERSION env variable
+        run: echo "EXIST_VERSION=${{ matrix.exist-version }}" >> $GITHUB_ENV
```

Comment:
if i'm not mistaken, [this](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsenv) would be preferable in the last step.